### ISSUE TITLE
Suggestion: User Participation with SteemConnect

### ISSUE THREAD

**Issue by clayop** [steemit/tinman#139]
Normal user's participation is very important to testnet but hard to attract their participation due to technical and attentional budgets. So I suggest mirroring their activities based on SteemConnect.

1. Steemit devs create a testnet account (e.g. steemtestnet)
2. Users register the testnet account to their posting authority via SteemConnect
3. The testnet account mirrors users' mainnet activities and collect error logs.
4. Some bounty is desired

**Comment by drov0** [comment]
Why not, we could have something partenered up with utopian.io. But we'll never be able to really mirror the load of the real blockchain. I suggest making a testnet that mirrors the current state of the main net (in terms of holdings for each account) and "simply" (I use that lightly there's probably a lot of hurdles along the way) stream the main net and reproduce all the operations that happen on the main net  on the testnet. 

Once we have that we can monitor the health of the network itself and instantly see how changes would behave in the main net. (RC costs for instance) and have a few human testers to make sure that we don't miss anything.  (Like they could try to abuse discount account creation).

Basically have bounties for testers where they are paid assuming they find a bug or a weird behavior.

**Comment by inertia186** [comment]
Sounds like a great idea.  Wider incentives to use testnet would be wonderful.  But I'm worried it would raise both signal and noise at the same time, leading back to the same problem, only now you're paying for it.

Keep in mind, we've been automatically copying activity from mainnet to testnet since Day 1.  So, don't re-invent the wheel:

https://github.com/steemit/tinman#gatling-transactions-from-mainnet

It's not 100% because of various issues, to name a few:

* Since everyone's balance has to fit into a total supply of 200,000,000 TESTS, transfers initiated on mainnet have a larger impact when repeated on testnet.
* Not every mainnet account exists on testnet (for various reasons), therefore not every post can be copied.
* Various other interruptions (like upstream errors and timeouts) cause operations to be lost, which leads to more operations being lost if they rely on them.

Also see: https://github.com/steemit/tinman/issues?utf8=%E2%9C%93&q=is%3Aissue+gatling
